src/test/resources/samples/1.22/info/2.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
:+1: Hard work, i spent a lot of time filling and comparing all this things initially.